README.md

```diff
@@ -31,6 +31,10 @@ Modules in `.spec.js` and `.test.js` are added to `devDependencies`
 
 `--yarn`    Use [yarn](https://yarnpkg.com) instead of npm
 
+`--install-types`  Tries to install `@types` type definitions for modules that don't have them packaged
```

Comment:
I would assume this by default without adding a flag. if it's a ts file, you most probably also want the types for it

Reply:
Hm... I see
It'd change how things are done, tbh
The way this is implemented rn types are installed by module, and not by file.
Maybe we could enable this by default if a `tsconfig.json` exists?

Reply:
Is this OK? To always install types if we find a `tsconfig.json`?

Reply:
I'd assume if you're using a .ts file, you'd also want types for your dependencies. Let me know if that assumption is wrong.

tsconfig might not be the right thing to depend on if you're using a framework/tool that hides that implementation detail